Dockerfile

```diff
@@ -20,8 +20,10 @@ RUN apt-get update && apt-get -y -q --no-install-recommends install wget \
     pipenv \
     && rm -rf /var/lib/apt/lists/*
 
-RUN mkdir /app
+RUN mkdir /var/www
```

Comment:
currently we do not run with docker. Should we remove this until it is finished?